lib/OpenQA/Markdown.pm

```diff
@@ -25,30 +19,46 @@ sub is_light_color {
     return $sum > 380;
 }
 
-sub _bugref_to_html ($bugref) {
-    my $bugurl = bugurl($bugref);
-    return "<a href=\"$bugurl\">$bugref</a>";
+sub bugref_to_html ($bugref, $fancy = 0) {
+	my $app = OpenQA::App->singleton;
```

Comment:
Looks like a `use OpenQA::App;` is missing in that file. (And despite the update this still needs tidying. We don't use tabs here.)

Reply:
You've updated the changes so this is no longer irrelevant; see my other comments regarding this problematic dependency structuring.